src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -544,7 +554,8 @@ def _to_vespa_hybrid_query(self, marqo_query: MarqoHybridQuery) -> Dict[str, Any
             'searchChain': 'marqo',
             'yql': 'PLACEHOLDER. WILL NOT BE USED IN HYBRID SEARCH.',
             'ranking': common.RANK_PROFILE_HYBRID_CUSTOM_SEARCHER,
-
+            'ranking.rerankCount': marqo_query.limit,       # limits the number of results going to phase 2
```

Comment:
would this impact recall? without this limit, how many results goes to phase 2?

Reply:
After talking to the vespa team, `ranking.rerankCount` will not shave off any results, but only pass a certain number to 2nd phase for ranking. All results not passed to 2nd phase will still be in the final result, just with a low score, effectively letting them be dropped at the end.

Regarding relevance, this is the behavior we want right? We only want the top N results out of phase 1 to be considered in phase 2.

Reply:
Let me use an example to elaborate my point. Let's say user has only one shard in Vespa. User queries with limit set to 10. The query is executed in that shard, retrieved 100 docs, first phase ranking ranked all 100 docs. Let's assume the first phase ranking does not do a very good job, some docs with high relevance falls out of top 10 results. We expect second phase ranking to improve the result. 
With the default setting, all 100 docs will be included in the second phase ranking. But with 'ranking.rerankCount' set to 10, only the top 10 docs will be included in the second phase ranking, which means we do not have a chance to rectify other docs' scores, leading to lower relevance in the returned result.